### ISSUE TITLE
Sitemap Generated in the Admin Panel is wrongly formatted

### ISSUE THREAD

**Issue by Dinesh-Ramakrishnan** [gocart/GoCart3#39]
I generated the sitemap using the sitemap option in the Admin Panel.  But the xml file is wrongly formatted. 

'<urlset>' opening tag is missing and it is placed in the place of closing tag.

While accessing it online I got the following error. 

```
This page contains the following errors:

error on line 8 at column 5: Extra content at the end of the document
Below is a rendering of the page up to the first error.

http://www.bubbleoutlooks.com/category/anklets 2016-02-16
```

I have attached the generated xml for your reference.

**Comment by Dinesh-Ramakrishnan** [comment]
```
<url>
    <loc>http://www.bubbleoutlooks.com/category/anklets</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/bangles</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/bracelets</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/bridalsets</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/chainsets</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/earrings</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/hairset</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/mangtikka</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/necklace</loc>
    <lastmod>2016-02-16</lastmod>
</url>


<url>
    <loc>http://www.bubbleoutlooks.com/category/pendants</loc>
    <lastmod>2016-02-16</lastmod>
</url>
```

<urlset>

**Comment by skynetweb** [comment]
Hi there, 
We fixed the sitemap module, you can do this by editing the file application/modules/sitemap/views/admin/sitemap_xml_foot.php and change `<urlset>` with `</urlset>`

Regards,
Alex

**Comment by PRDinesh** [comment]
fixed with commit : 7933e9fc381ce5e46d47d39064acb76e01f3e71f
